### PR TITLE
script: Fix issues relating to <iframe frameborder>.

### PR DESCRIPTION
**Pull request by pcwalton**
These changes implement the `frameBorder` attribute on `HTMLIFrameElement` and fix the syntax of the rule that implements this presentational hint in the presentational hint stylesheet.

Improves google.com.

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/13926)

### CODE REVIEW

**Comment by jdm** [comment]
See what happens when you ignore the checklist? ;)
```
36.36s$ ./mach test-tidy --no-progress --all
Checking the config file...
Checking directories for correct file extensions...
Checking files for tidiness...
./components/script/dom/htmliframeelement.rs:547: link to WHATWG may break in the future, use this format instead: https://html.spec.whatwg.org/multipage/#other-elements
./components/script/dom/htmliframeelement.rs:549: link to WHATWG may break in the future, use this format instead: https://html.spec.whatwg.org/multipage/#other-elements
```

File: components/script/dom/htmliframeelement.rs [hunk 1]
```diff
@@ -544,6 +544,11 @@ impl HTMLIFrameElementMethods for HTMLIFrameElement {
     // https://html.spec.whatwg.org/multipage/#dom-dim-height
     make_dimension_setter!(SetHeight, "height");
 
+    // https://html.spec.whatwg.org/multipage/obsolete.html#other-elements,-attributes-and-apis:attr-iframe-frameborder
+    make_getter!(FrameBorder, "frameborder");
+    // https://html.spec.whatwg.org/multipage/obsolete.html#other-elements,-attributes-and-apis:attr-iframe-frameborder
```
Comment (emilio) [inline components/script/dom/htmliframeelement.rs]: I think the multipage/foo.html links are discouraged, and multipage/  only is preferred.

**Review by emilio — APPROVED** [review]
r=me with the spec link changed if possible.

**Comment by emilio** [comment]
Oh, of course I didn't read the previous comment by jdm :)

**Comment by pcwalton** [comment]
Fixed. r? @jdm

**Comment by jdm** [comment]
```
  ▶ Unexpected subtest result in /html/dom/interfaces.html:
  └ PASS [expected FAIL] HTMLIFrameElement interface: attribute frameBorder

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: typeof IDL attribute

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL get with DOM attribute unset

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to "" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to " \\0\\x01\\x02\\x03\\x04\\x05\\x06\\x07 \\b\\t\\n\\v\\f\\r\\x0e\\x0f \\x10\\x11\\x12\\x13\\x14\\x15\\x16\\x17 \\x18\\x19\\x1a\\x1b\\x1c\\x1d\\x1e\\x1f  foo " followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to undefined followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to 7 followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to 1.5 followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to true followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to false followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to object "[object Object]" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to NaN followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to Infinity followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to -Infinity followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to "\\0" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to null followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to object "test-toString" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: setAttribute() to object "test-valueOf" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to "" followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to " \\0\\x01\\x02\\x03\\x04\\x05\\x06\\x07 \\b\\t\\n\\v\\f\\r\\x0e\\x0f \\x10\\x11\\x12\\x13\\x14\\x15\\x16\\x17 \\x18\\x19\\x1a\\x1b\\x1c\\x1d\\x1e\\x1f  foo " followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to undefined followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to undefined followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to 7 followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to 7 followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to 1.5 followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to 1.5 followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to true followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to true followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to false followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to false followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to object "[object Object]" followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to object "[object Object]" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to NaN followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to NaN followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to Infinity followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to Infinity followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to -Infinity followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to -Infinity followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to "\\0" followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to null followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to null followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to object "test-toString" followed by getAttribute()

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to object "test-toString" followed by IDL get

  ▶ Unexpected subtest result in /html/dom/reflection-embedded.html:
  └ PASS [expected FAIL] iframe.frameBorder: IDL set to object "test-valueOf" followed by IDL get
```

**Pull request merged**